### PR TITLE
LPS-103601 Create Account Wizard for creating a new Account User

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-103601

@jonmak08

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/select_account_entry.jsp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+%>
+
+<clay:management-toolbar
+	displayContext="<%= selectAccountEntryManagementToolbarDisplayContext %>"
+	showCreationMenu="<%= false %>"
+/>
+
+<aui:container cssClass="container-fluid container-fluid-max-xl">
+	<aui:form method="post" name="fm">
+		<aui:input name="accountEntryIds" type="hidden" />
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/select_account_entry.jsp]: What are these used for?

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/select_account_entry.jsp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+		<liferay-ui:search-container
+			searchContainer="<%= accountEntryDisplaySearchContainer %>"
+		>
+			<liferay-ui:search-container-row
+				className="com.liferay.account.admin.web.internal.display.AccountEntryDisplay"
+				keyProperty="accountId"
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/select_account_entry.jsp]: This has been renamed to `accountEntryId` for the AccountEntryDisplay

**Comment by alee8888** [comment]
Changes resent in https://github.com/pei-jung/liferay-portal/pull/535

**Pull request closed without merging**